lib/checks/canMultiplyLikeTermConstantNodes.js

```diff
@@ -0,0 +1,63 @@
+const Node = require('../node');
+const NodeType = require('../node/Type');
+
+// Returns true if node is a multiplication of constant power nodes
+// where you can combine their exponents, e.g. 10^2 * 10^4 * 10 can become 10^7
+
+function canMultiplyLikeTermConstantNodes(node) {
+  if (!Node.Type.isOperator(node) || node.op !== '*') {
+    return false;
+  }
+  const args = node.args;
+  if (!args.every(n => isPowerTerm(n))) {
```

Comment:
`isConstantOrConstantPower`? hmm that's kinda long, but I think adding more detail to the name would be better. maybe you can think of something with more detail but not that long (if you don't like my name)